libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -152,6 +157,32 @@ class DropInNavigationView @JvmOverloads constructor(
     private inline fun <reified T : ViewModel> lazyViewModel(): Lazy<T> = lazy {
         viewModelProvider[T::class.java]
     }
+
+    private fun startTripSession() {
```

Comment:
Will you encapsulate this into a component or view model. We don't want to add logic to `DropInNavigationView `

Reply:
The requirement in the ticket request this to be done in DropInNavigationView.